core/src/test/java/io/jenkins/pluginhealth/scoring/probes/CodeCoverageProbeTest.java

```diff
@@ -49,16 +49,16 @@
 import org.kohsuke.github.PagedIterable;
 
 class CodeCoverageProbeTest extends AbstractProbeTest<CodeCoverageProbe> {
-    @Override
-    CodeCoverageProbe getSpy() {
-        return spy(CodeCoverageProbe.class);
-    }
-
     @Test
     public void shouldNotRequireRelease() {
         assertThat(getSpy().requiresRelease()).isFalse();
     }
 
+    @Override
+    CodeCoverageProbe getSpy() {
+        return spy(CodeCoverageProbe.class);
+    }
+
```

Comment:
Why moving methods around?

Reply:
It is automated formatting. That is why I did not change it.